hardware/detect.py

```diff
@@ -726,6 +687,86 @@ def find_element(xml, xml_spec, sys_subtype,
     return True
 
 
+def get_cpus(hw_lst, lscpu_output=None, lscpux_output=None, scaling_governor_output=None):
+    # Extracting lspcu information
+    lscpu = {}
+    if lscpu_output is None:
+        output = detect_utils.output_lines('LANG=en_US.UTF-8 lscpu')
+    else:
+        output = lscpu_output.splitlines()
+    for line in output:
+        item, value = line.split(':')
```

Comment:
Do you want to limit the number of splits like `line.split(':', 1)`?

Reply:
Btw `lscpu` seems to have `--json` flag, maybe it's easier to use?

Reply:
The json is pretty crappy as it has columns in the structure... and it could have any comas.